actors/abi/sector.go

```diff
@@ -207,11 +207,13 @@ func (p RegisteredSealProof) RegisteredWindowPoStProof() (RegisteredPoStProof, e
 	}
 }
 
-// SectorMaximumLifetime is the maximum duration a sector sealed with this proof may exist between activation and expiration
+// The maximum duration a sector sealed with this proof may exist between activation and expiration.
+// This ensures that SDR is secure in the cost model for Window PoSt and in the time model for Winning PoSt
+// This is based on estimation of hardware latency improvement and hardware and software cost reduction.
 func (p RegisteredSealProof) SectorMaximumLifetime() ChainEpoch {
 	// For all Stacked DRG sectors, the max is 5 years
-	epochsPerYear := 1_262_277
-	fiveYears := 5 * epochsPerYear
+	epochsPerYear := 1_051_200 // 365 days / 30 seconds
+	fiveYears := 5 * epochsPerYear // PARAM_SPEC
```

Comment:
Ideally PARAM_SPEC is at the beginning of a comment of a variable for easy parsing for the spec.

Reply:
If it's on a new line above, grep output becomes much harder to parse.

E.g. this is now almost useful:
```
[~/p/f/specs-actors](param-audit)$ rg 'PARAM_SPEC'
actors/builtin/network.go
9:// PARAM_SPEC
25:// PARAM_SPEC

actors/abi/sector.go
216:	fiveYears := 5 * epochsPerYear // PARAM_SPEC

actors/builtin/power/policy.go
10:const ConsensusMinerMinMiners = 4 // PARAM_SPEC
20:var ConsensusMinerMinPower = abi.NewStoragePower(100 << 40) // PARAM_SPEC PARAM_FINISH
26:const MaxMinerProveCommitsPerEpoch = 200 // PARAM_SPEC

actors/builtin/market/policy.go
10:const DealUpdatesInterval = builtin.EpochsInDay // PARAM_SPEC
15:	Numerator: big.NewInt(5), // PARAM_SPEC
22:var DealMinDuration = abi.ChainEpoch(180 * builtin.EpochsInDay) // PARAM_SPEC
25:var DealMaxDuration = abi.ChainEpoch(540 * builtin.EpochsInDay) // PARAM_SPEC

actors/builtin/miner/monies.go
14:var PreCommitDepositFactor = 20 // PARAM_SPEC PARAM_FINISH
19:var InitialPledgeFactor = 20 // PARAM_SPEC PARAM_FINISH
25:	Numerator:   big.NewInt(3), // PARAM_SPEC PARAM_FINISH
34:var DeclaredFaultFactorNum = 214 // PARAM_SPEC
44:var UndeclaredFaultProjectionPeriod = abi.ChainEpoch(5) * builtin.EpochsInDay // PARAM_SPEC

actors/builtin/miner/policy.go
18:var WPoStProvingPeriod = abi.ChainEpoch(builtin.EpochsInDay) // 24 hours PARAM_SPEC
23:var WPoStChallengeWindow = abi.ChainEpoch(30 * 60 / builtin.EpochDurationSeconds) // 30 minutes (48 per day) PARAM_SPEC
27:const WPoStPeriodDeadlines = uint64(48) // PARAM_SPEC
53:const AddressedSectorsMax = 10_000 // PARAM_SPEC
59:	MaxPeerIDLength = 128 // PARAM_SPEC
62:	MaxMultiaddrData = 1024 // PARAM_SPEC
79:const ChainFinality = abi.ChainEpoch(1400) // PARAM_SPEC
99:	abi.RegisteredSealProof_StackedDrg32GiBV1:  abi.ChainEpoch(10000), // PARAM_SPEC PARAM_FINISH
109:var MaxPreCommitRandomnessLookback = abi.ChainEpoch(10000) // PARAM_SPEC
114:var PreCommitChallengeDelay = abi.ChainEpoch(150) // PARAM_SPEC
120:const WPoStChallengeLookback = abi.ChainEpoch(20) // PARAM_SPEC
126:const FaultDeclarationCutoff = WPoStChallengeLookback + 50 // PARAM_SPEC
130:var FaultMaxAge = WPoStProvingPeriod * 14 // PARAM_SPEC
134:const WorkerKeyChangeDelay = ChainFinality // PARAM_SPEC
137:const MinSectorExpiration = 180 * builtin.EpochsInDay // PARAM_SPEC
142:const MaxSectorExpirationExtension = 540 * builtin.EpochsInDay // PARAM_SPEC
147:const DealLimitDenominator = 134217728 // PARAM_SPEC
204:	Numerator:   big.NewInt(1), // PARAM_FINISH PARAM_SPEC
210:	Numerator:   big.NewInt(101251), 	// PARAM_FINISH PARAM_SPEC
216:	Numerator:   big.NewInt(1), // PARAM_SPEC
229:var RewardVestingSpec = VestSpec{ // PARAM_SPEC
```